### ISSUE TITLE
"Oops! Something went wrong." only taking up part of screen

### ISSUE THREAD

**Issue by fsih** [LLK/scratch-gui#2246]
![image](https://user-images.githubusercontent.com/2855464/40987546-c3b23050-68b6-11e8-9aca-97d650d5cfbf.png)

Should it fill the area it's replacing, or take up the entire screen?

**Comment by amazinigmech2418** [comment]
What do you mean "Use an undefined variable somewhere in scratch-paint"?

**Comment by fsih** [comment]
I don't know how you'd reproduce this bug on https://llk.github.io/scratch-gui/develop/
I got it by running a local copy of the code and changing the code to use an undeclared variable.

**Comment by amazinigmech2418** [comment]
@fsih Maybe, the local script changes caused the bug...

**Comment by fsih** [comment]
Yes, the local change caused the error screen to appear. But the error screen shouldn't look like that, that's the issue.

**Comment by amazinigmech2418** [comment]
The issue wouldn't be able to be caused in the editor, so it isn't really a problem.

**Comment by fsih** [comment]
It's a problem since breaking bugs sometimes end up in develop by accident. That's why an error screen exists.

**Comment by amazinigmech2418** [comment]
I thought that was for if a person added an undefined block to the editor.

**Comment by fsih** [comment]
Sorry, I think I said it in a confusing way. If you mess up the Javascript that makes up Scratch, then it's supposed to make the error screen appear. One of the ways to mess up the Javascript is to use undefined Javascript variables, like if you replace the line
`const PaintEditorComponent = props => (`
with 
`const PaintEditorComponent = () => (`
but there are lots of other ways.
